lib/view_component/form/helpers/rails.rb

```diff
@@ -19,6 +19,7 @@ def self.included(base)
               label
               phone_field
               radio_button
+              textarea
```

Comment:
I think we may keep it at auto generate like it was. Like for `phone_field` and `telephone_field`.

And so removing the `textarea` method bellow, and moving the alias just after `alias_method :phone_field, :telephone_field`.